cypress/e2e/real-time-events/advanced-filter.cy.js

```diff
@@ -1,21 +1,21 @@
 /* eslint-disable cypress/no-unnecessary-waiting */
 import selectors from '../../support/selectors'
 
-describe('Real-Time Events Advanced Filter', { tags: ['@dev3'] }, () => {
+describe('Real-Time Events Advanced Filter', { tags: ['@dev3', '@skip'] }, () => {
   beforeEach(() => {
     cy.login()
     cy.openProduct('Real-Time Events')
     cy.intercept('POST', '/v4/events/graphql').as('getGraphqlFilters')
     cy.get(`${selectors.realTimeEvents.filterOptions} > [tabindex="-1"]`).click()
   })
 
-  it('should open advanced filter', () => {
+  it.skip('should open advanced filter', () => {
```

Comment:
I don't get it. All these skips ...
The change on line 4 isn't there to skip the it.skips? Or do you want to add a "marker"?